common/persistence/dataInterfaces.go

```diff
@@ -113,6 +113,11 @@ type (
 		Msg string
 	}
 
+	// AppendHistoryError represents a failed insert to history tree / node request
+	AppendHistoryError struct {
+		Msg string
```

Comment:
AppendHistoryTimeoutError maybe?

Please also update the logic in persistence metrics client (the switch case).